scylla-persistence/src/main/java/com/netflix/conductor/scylla/dao/ScyllaExecutionDAO.java

```diff
@@ -1262,9 +1279,18 @@ public void setConcurrencyLimitEnabled(boolean concurrencyLimitEnabled) {
         this.isConcurrencyLimitEnabled = concurrencyLimitEnabled;
     }
 
+    public boolean isFallbackToOldTaskInProgressActive() {
+        return fallbackToOldTaskInProgressActive;
+    }
+
+    public void setFallbackToOldTaskInProgressActive(boolean fallbackToOldTaskInProgressActive) {
+        this.fallbackToOldTaskInProgressActive = fallbackToOldTaskInProgressActive;
+    }
+
     @Override
     public void setApplicationContext(ApplicationContext applicationContext) throws BeansException {
         this.redisLock = (RedisLock) applicationContext.getBean("provideRedisLock");
         setConcurrencyLimitEnabled(Boolean.parseBoolean(applicationContext.getEnvironment().getProperty("concurrencyLimitEnabled", "false")));
+        setFallbackToOldTaskInProgressActive(Boolean.parseBoolean(applicationContext.getEnvironment().getProperty("fallbackToOldTaskInProgressActive", "false")));
```

Comment:
make sure the secret is added with true. also can we make true as default value.

Reply:
will add secret as true, this will be short living flag for migration only.